### PR TITLE
[15.0][MIG] product_supplierinfo_for_customer_sale

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Based on product_supplierinfo_for_customer, this module loads in every sale order the
customer code defined in the product and allows use the product codes and product name
configured in each products in sale orders.

If you use Advanced price rules with formulas to define your pricing, and
choose that the price should be calculated from the partner prices in the
product form, the quantity in the sales order will be proposed from
the minimum quantity defined in the customerinfo.

### CODE REVIEW

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: ForgeFlow

File: product_supplierinfo_for_customer_sale/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2017 Agile Business Group sagl
+#     (<http://www.agilebg.com>)
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Product Supplierinfo for Customer Sale",
+    "version": "15.0.1.0.1",
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_for_customer_sale/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by cesar-tecnativa — APPROVED** [review]

File: product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py (hunk 1 above)
Comment (JuanyDForgeflow) [reply product_supplierinfo_for_customer_sale/tests/test_product_supplierinfo_for_customer_sale.py]: Done

File: product_supplierinfo_for_customer_sale/__manifest__.py (hunk 2 above)
Comment (JuanyDForgeflow) [reply product_supplierinfo_for_customer_sale/__manifest__.py]: Done

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@JuanyDForgeflow Can you add this commit https://github.com/OCA/sale-workflow/commit/53997acfb0440fd662153be862fe1d4c7d6e8873

Thanks!!

**Comment by JuanyDForgeflow** [comment]
@sergio-teruel I added the commit. Thanks

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**